### ISSUE TITLE
Make Precaution scan the whole repository

### ISSUE THREAD

**Issue by MVrachev** [vmware/precaution#120]
**Is your feature request related to a problem? Please describe.**

It would be really cool if Precaution scanned the whole repository upon installation. That way they can fix their bugs before scanning new pull request.

**Describe the solution you'd like**

An option to scan the whole pull request multiple times not only once at the beginning. It would be great if there is a way to see the problems next to the lines as it is done through the Checks API.
